client/components/marketing-survey/cancel-purchase/index.jsx

```diff
@@ -0,0 +1,357 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import shuffle from 'lodash/shuffle';
+
+/**
+ * Internal Dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+
+const CancelPurchaseSurvey = React.createClass( {
```

Comment:
Yeah, I have linting on, just wasn't sure about what to do about naming since `CancelPurchase` was a little too generic.

I've moved the file to `cancel-purchase-form/index.jsx`.